assets/js/system-status.ts

```diff
@@ -77,14 +66,14 @@ window.addEventListener('DOMContentLoaded', () => {
     // page only once, whereas the other icons show twice.
     // All are hidden on page load using display: none;.
 
-    if ($icons.length > 1) {
+    if ($status.icon.length > 1) {
       $headerStatus.classList.add('show-header-system-status');
-      $headerStatusMessage.innerText = message;
+      $headerStatusMessage.innerText = `${$status.title}: ${$status.message}`;
     } else {
       $headerStatus.classList.remove('show-header-system-status');
     }
 
-    $icons.forEach((icon) => {
+    $status.icon.forEach((icon) => {
```

Comment:
I think this helps illustrate the comment above where `icons` might be more suitable variable name. If you prefer to keep `icon`, then I suggest maybe the following so that the name `icon` isn't used twice:

```
$status.icon.forEach((i) => {
  (i as Element).classList.add('show-status-icon');
});
```

Reply:
I just thought it might be less confusing to keep the types similar between sites, but I see that it really is a collection of icons, so the plural makes sense. 👍🏻